skimage/transform/hough_transform.py

```diff
@@ -315,6 +317,12 @@ def hough_circle_peaks(hspaces, radii, min_xdistance=1, min_ydistance=1,
     >>> img[x, y] = 1
     >>> hspaces = transform.hough_circle(img, radius)
     >>> accum, cx, cy, rad = hough_circle_peaks(hspaces, [radius,])
+
+    Notes
+    -----
+    Circles with bigger radius have higher peaks in Hough space. If larger
+    circles are preferred over smaller ones, `normalize` should be False.
+    Otherwise, circles will be returned in the order of decreasing perfectness.
```

Comment:
I'm not a huge fan of "perfectness" here. How about "voting number" (this is the terminology used in https://en.wikipedia.org/wiki/Circle_Hough_Transform).

Reply:
agreed and changed.